pegasus/sites.v3/code.org/public/css/homepage.css

```diff
@@ -373,3 +373,28 @@ body.modal-open .supreme-container{
   color: rgb(82, 82, 82);
   border: 1px solid rgb(146,147,148);
 }
+
```

Comment:
@wjordan  I included you because I know you and Hema did all of that great optimization work on the homepage css.  Should I have added these styles here? Is there a better way?